style.css

```diff
@@ -0,0 +1,59 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;600;700;800&display=swap");
+
+@media only screen and (max-device-width: 375px) {
+  div#wrapper {
+    width: 400px;
+  }
+}
+
+div {
+  width: 35px;
+  height: 2px;
+  background-color: #172b43;
+  margin: 4px 0;
+  border-radius: 30%;
+  border: 1px solid #172b43;
+}
+
+.menu_icon {
+  padding-left: 85%;
+  padding-right: 30%;
+}
+
+.heading_up {
+  color: #172b4d;
+  text-align: center;
+  font-family: Inter, sans-serif;
+  font-size: 40px;
+}
+
+.heading_down {
+  color: #36b37e;
+  text-align: center;
+  font-family: Inter, sans-serif;
+  font-size: 40px;
+}
+
+.main_section {
+  background-image: url("img/header-illsutration-mobile@2x.png");
+  background-size: 100% auto;
+  background-position: center center;
+}
+
+p {
+  text-align: center;
+  font-family: Inter, sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  margin: 8%;
+}
+
+.social_links {
+  margin-left: 80px;
+  margin-right: 80px;
+  color: #505f79;
+}
+
+.icon {
+  margin: 7px;
+}
```

Comment:
kindly implement the following design changes to enable you have a better UI and good user interaction:

- [ ] Your hamburger menu has a bigger width than what we have in the design given, please change it so we have a similar design with yours
- [ ] The `font-color` property of your paragraph is different from what we have in the design given, kindly use this         
    ` #172B4D` color for your font in that paragraph

figma design :
![image](https://github.com/ishimwepati/Portfolio/assets/20537712/638318f1-f851-4261-a6a2-d473c0df0cb0)

Your design :
![image](https://github.com/ishimwepati/Portfolio/assets/20537712/591246f1-654b-441a-ba54-cbb892445e08)

Reply:
Wow, thanks for this guidances.